test/basics/demo-complex.html

```diff
@@ -0,0 +1,18 @@
+<script type='text/stache' can-autorender id='demo-html'>
+	{{#each items}}
+		<li>{{name}}</li>
+	{{/each}}
+</script>
+</script>
+
+<script src="../../node_modules/steal/steal.js" main="@empty" id='demo-source'>
+import "can-view-autorender";
+import "can-stache-bindings";
```

Comment:
It seems excessive to bring in so many dependencies for a test. Is this one needed? Are there any others that could be removed?